camdecmpswks/procedures/refresh_emission_view_matshg.sql

```diff
@@ -25,43 +25,80 @@ BEGIN
-                              on ems.rpt_period_id = par.rpt_period_id
-                             and ems.mon_plan_id = par.mon_plan_id
-                            join camdecmpsmd.REPORTING_PERIOD prd
-                              on prd.rpt_period_id = ems.rpt_period_id
-                            join camdecmpswks.MONITOR_PLAN_LOCATION mpl
-                              on mpl.mon_plan_id= ems.mon_plan_id 
-                            join camdecmpswks.SORBENT_TRAP trp
-                              on trp.rpt_period_id = ems.rpt_period_id 
-                             and trp.mon_loc_id = mpl.mon_loc_id
-                             and trp.begin_date <= prd.end_date
-                             and trp.end_date >= prd.begin_date
-                            join camdecmpswks.SAMPLING_TRAIN trn
-                              on trn.rpt_period_id = ems.rpt_period_id 
-                             and trn.trap_id = trp.trap_id
+                                select  trp.mon_loc_id,
+                                        trp.mon_sys_id,
+                                        trp.begin_date,
+                                        trp.begin_hour,
+                                        trp.end_date,
+                                        trp.end_hour,
+                                        trn.component_id
+                                  from  (
+                                            select  vmonplanid as mon_plan_id,
+                                                    vrptperiodid as rpt_period_id
+                                        ) par
+                                        join camdecmpswks.EMISSION_EVALUATION ems
+                                          on ems.rpt_period_id = par.rpt_period_id
+                                         and ems.mon_plan_id = par.mon_plan_id
+                                        join camdecmpsmd.REPORTING_PERIOD prd
+                                          on prd.rpt_period_id = ems.rpt_period_id
+                                        join camdecmpswks.MONITOR_PLAN_LOCATION mpl
+                                          on mpl.mon_plan_id= ems.mon_plan_id 
+                                        join camdecmpswks.SORBENT_TRAP trp
+                                          on trp.rpt_period_id = ems.rpt_period_id 
+                                         and trp.mon_loc_id = mpl.mon_loc_id
+                                         and trp.begin_date <= prd.end_date
+                                         and trp.end_date >= prd.begin_date
+                                        join camdecmpswks.SAMPLING_TRAIN trn
+                                          on trn.rpt_period_id = ems.rpt_period_id 
+                                         and trn.trap_id = trp.trap_id
+                                        join camdecmpswks.COMPONENT cmp
+                                          on cmp.component_id = trn.component_id

```

Comment:
This join is not in the corresponding CAMDECMPS procedure, should it be? It doesn't look like it's necessary.

Reply:
Good catch Max!  Removing, will retest, and request another review.